### PR TITLE
Create installing-and-using-lets-encrypts-certbot-utility-on-centos7-…

### PR DESCRIPTION
**Pull request by BadCrumpany**
…with-apache

### CODE REVIEW

**Comment by catlook** [comment]
build me

**Comment by catlook** [comment]
build me

**Comment by catlook** [comment]
build me

**Comment by catlook** [comment]
build me

**Comment by robb-romans** [comment]
@catlook Check the Markdown source file for errors and incorrect syntax. The Slack error is a symptom, not a root cause.

**Comment by catlook** [comment]
build me

**Comment by catlook** [comment]
build me

**Comment by stephamon** [comment]
build me

File: content/cloud-servers/install-and-use-the-lets-encrypts-certbot-utility-on-centos7-with-apache.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+
+### What is Let's Encrypt?
+
+Let's Encrypt is a Certificate Authority (CA) that provides free 90-day SSL Certificates.  The following details
+come from their [About Page](https://letsencrypt.org/about/):
+
+> **Let's Encrypt is a free, automated, and open certificate authority (CA), run for the public's benefit. It is a service provided by the Internet Security Research Group (ISRG).**
+>
+> **We give people the digital certificates they need in order to enable HTTPS (SSL/TLS) for websites, for free, in the most user-friendly way we can. We do this because we want to create a more secure and privacy-respecting Web.**
+>
```
Comment (robb-romans) [inline content/cloud-servers/install-and-use-the-lets-encrypts-certbot-utility-on-centos7-with-apache.md:24]: Did you intend to have these `>` characters in this section?

Comment (stephamon) [reply content/cloud-servers/install-and-use-the-lets-encrypts-certbot-utility-on-centos7-with-apache.md:24]: Oh I'm not sure, I was testing a header fix. I'll pull those out too

Comment (stephamon) [reply content/cloud-servers/install-and-use-the-lets-encrypts-certbot-utility-on-centos7-with-apache.md:27]: It's supposed to be a blockquote, I discovered, so I fixed it a bit

Comment (catlook) [reply content/cloud-servers/install-and-use-the-lets-encrypts-certbot-utility-on-centos7-with-apache.md:27]: I don't think block quotes work for us.  At least the didn't during the recent blog repo test.

**Comment by catlook** [comment]
build me

**Comment by catlook** [comment]
build me

**Comment by WilliamLoy** [comment]
build me

**Comment by WilliamLoy** [comment]
build me

**Comment by rackernexus** [comment]
Build 2278 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-337c0da16/how-to/). valid for 14 days

**Comment by catlook** [comment]
build me

**Comment by rackernexus** [comment]
Build 2279 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-337c0da16/how-to/). valid for 14 days

**Comment by catlook** [comment]
build me

**Comment by rackernexus** [comment]
Build 2280 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-5f58fcd4b/how-to/). valid for 14 days

**Comment by catlook** [comment]
build me

**Comment by catlook** [comment]
build me

**Comment by rackernexus** [comment]
Build 2282 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-513f776ac/how-to/). valid for 14 days

**Comment by catlook** [comment]
shawn approved!

**Pull request merged**